src/components/Shared/TabbedContainer/Tab.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from "react";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+import styled from "@emotion/styled";
+import { STYLES } from "../../../constants/StyleConstants";
+
+const TabView = styled.div({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  flex: 1,
+  flexDirection: "row",
+  padding: 8,
+});
+
+type TabProps = {
+  /** when the user clicks this tab, this function executes */
+  setActive: (index: number) => void;
+  /** whether or not this tab is selected */
+  isActive: boolean;
+  /** the index of the tab */
+  index: number;
+  /**
+   * the component to display as tab labels.
+   * this can be a simple string or be an enclosed combination, e.g. an icon and string.
+   **/
+  children?: JSX.Element;
+};
```

Comment:
I forget where exactly we landed on this, but should this be an `interface` instead of a `type` to be consistent with the conventions of other components?

cc @tohuynh

Reply:
I believe we decided on it can be either.